_templates/build.yml.erb

```diff
@@ -30,7 +32,7 @@ jobs:
             tag: <%= tag %>
             target: <%= target %><% end %>
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
This change was needed because a prior change was made directly in `.github/workflows/build.yml` instead of the template. 

The project's file structure requires updates to the template and then running the `build.rb` script to regenerate the actual `build.yml`.